docs/src/content/howto-transparent-vms.md

```diff
@@ -14,7 +14,7 @@ Internal Network* setup can be applied to other setups.
 
 ## 1. Configure Proxy VM
 
-On the proxy machine, **eth0** is connected to the internet. **eth1** is
+On the proxy machine, **eth0** or **enp0s3** (Ubuntu 15.10 and newer) is connected to the internet. **eth1**  or **enp0s8** (Ubuntu 15.10 and newer) is
```

Comment:
The interface names are pretty much random at this point - and highly depend on your hardware.
`eth0` should be considered a placeholder here, so the user has to verify which interface name it really has.
Would you mind re-writing the sentence to make the user "find" the correct interface?

Reply:
Saying that the interface names are "pretty much random at this point" is not quite true. Actually, systemd calls them [Predictable Network Interface Names](https://www.freedesktop.org/wiki/Software/systemd/PredictableNetworkInterfaceNames/) and in this case, they're assigned by their "physical location".

Now, considering this tutorial covers VirtualBox, I'm fairly certain that enp0s3 maps to your first network interface and enp0s8 to the second one. Maybe one can create a different assignment by fiddling around with the virtual machine config? Might be the case, but I haven't seen this before. [And some other docus also cover it this way.](https://bertvv.github.io/notes-to-self/2015/09/29/virtualbox-networking-an-overview/).

Reply:
Yes, some systems might have predictable names, but IMHO there are too many systems / managers / OS / Linux distributions to have a "one size fits all" documentation.
I would not rely on any interface name that is written in the docs - the user should always double-check what their own system is using - especially in multi-homed servers / appliances.

Reply:
But the tutorial is specified to Ubuntu and VirtualBox, but yeah... I've updated the text to look for the correct names via `ip link`, might wanna take a look on the new version?